terraform/modules/rds/database.tf

```diff
@@ -8,21 +7,27 @@ resource "aws_db_instance" "this" {
   #checkov:skip=CKV_AWS_353:performance insights TODO CLDC-2660 if necessary
   #checkov:skip=CKV_AWS_354:performance insights TODO CLDC-2660 if insights are necessary
   #checkov:skip=CKV2_AWS_30:query logging TODO CLDC-2660
-  #checkov:skip=CKV2_AWS_60:copy tags to snapshots TODO CLDC-2661
   identifier                 = var.prefix
   apply_immediately          = true
   auto_minor_version_upgrade = true
   allocated_storage          = var.allocated_storage #units are GiB
+  backup_retention_period    = var.backup_retention_period
+  backup_window              = "23:09-23:39"
+  copy_tags_to_snapshot      = true
```

Comment:
We don't have any tags currently, but it seems wise to want this in case we do ever have tags.